PCbuild/rt.bat

```diff
@@ -55,7 +55,7 @@ echo on
 %cmd%
 @echo off
 
-echo About to run again without deleting .pyc/.pyo first:
+echo About to run again without deleting .pyc first:

```

Comment:
These changes in `rt.bat` are fine, please include them.  Leave `Tools/buildbot/clean.bat` alone, though.